getpaid_payu/client.py

```diff
@@ -140,11 +140,9 @@ def new_order(
             "description": description if description else "Payment order",
             "currencyCode": currency,
             "totalAmount": self._convert(amount),
-            "products": self._normalize(products)
-            if products
-            else [
+            "products": self._normalize(products) if products else [
                 {
-                    "name": "Total order",
+                    "name": "Wireless mouse",
```

Comment:
This is the fallback name of the order, don't change it.

Reply:
But "Total order" can suggest wrong purpose. In my opinion good name is "Name of product" etc